Documentation/ExtensionArchitecture/CreateNewExtension/Index.rst

```diff
@@ -23,36 +23,16 @@ Kickstarting the extension
 ==========================
 
 Although it is possible to write every single line of an extension from
-scratch, there is a tool which makes it easier to start. It is called
-"Extension builder" (key: "extension_builder") and can be installed from
-TER.
+scratch, there is a tool which makes it easier to start: The
+`Extension Builder <https://extensions.typo3.org/extension/extension_builder>`_.
 
-.. hint::
+The Extension Builder comes with its own BE module and helps you to create the
```

Comment:
BE - backend

Reply:
This is the original text: i just extracted it and backported it. We should adapt such terminologies in a separate run: there are many more to adapt.